pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -221,6 +221,14 @@ const (
 	VolumeTypeST1 = "st1"
 )
 
+// AWS provisioning limits.
+// Source: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html
+const (
+	MaxIOPSPerGB = 50
+	MinTotalIOPS = 100
+	MaxTotalIOPS = 20000
```

Comment:
IOPS can be set by user only for `io1` volumes, IOPS for the rest of volumes is preset by AWS.